draft-ietf-quic-recovery.md

```diff
@@ -1142,6 +1142,22 @@ Pseudocode for OnPacketSent follows:
      SetLossDetectionTimer()
 ~~~
 
+## On Receiving a Datagram
+
+When a server is blocked by anti-amplification limits, receiving
+a datagram unblocks it, even if none of the packets in the
+datagram are successfully processed. In such a case, the PTO
+timer may need to be armed.
```

Comment:
When would it not be armed?

Reply:
I thought a padded ACK-only packet could hit the limit without bytes in flight, but I realize that each full-sized ACK allows 3 packets to be sent.  Updating text.